<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>632</width>
    <height>421</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>218</width>
     <height>284</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Z:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="CXDoubleLabel" name="bpm_xstd"/>
    </item>
    <item row="3" column="3">
     <widget class="CXDoubleLabel" name="bpm_zstd"/>
    </item>
    <item row="2" column="1">
     <widget class="CXDoubleLabel" name="bpm_x"/>
    </item>
    <item row="3" column="2">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Z_STD:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="2">
     <widget class="QLabel" name="bpm_name">
      <property name="text">
       <string>BPM N</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="CXSwitch" name="bpm_is_on"/>
    </item>
    <item row="0" column="0">
     <widget class="CXEventLed" name="bpm_event"/>
    </item>
    <item row="3" column="1">
     <widget class="CXDoubleLabel" name="bpm_z"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>X:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>X_STD:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="4">
     <layout class="QGridLayout" name="view_grid"/>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CXSwitch</class>
   <extends>QPushButton</extends>
   <header>cxwidgets</header>
  </customwidget>
  <customwidget>
   <class>CXEventLed</class>
   <extends>LedWidget</extends>
   <header>cxwidgets</header>
  </customwidget>
  <customwidget>
   <class>CXDoubleLabel</class>
   <extends>QLabel</extends>
   <header>cxwidgets</header>
  </customwidget>
  <customwidget>
   <class>LedWidget</class>
   <extends>QWidget</extends>
   <header>cxwidgets</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
